docs/book/src/topics/identities-use-cases.md

```diff
@@ -1,45 +1,41 @@
-## How to use Identities with CAPZ
+# How to use Identities with CAPZ
 
-## CAPZ controller: 
-This is the identity used by the management cluster to provision infrastructure in Azure
- - Multi-tenant config via AzureClusterIdentity
-   - [AAD Pod Identity](https://azure.github.io/aad-pod-identity/) using Service Principals and Managed Identities: by default, the identity used by the workload cluster running on Azure is the same Service Principal assigned to the management cluster. If an identity is specified on the Azure Cluster Resource, that identity will be used when creating Azure resources related to that cluster. See [Multi-tenancy](multitenancy.md) page for details.
-More details in [Azure built-in roles documentation](https://learn.microsoft.com/azure/role-based-access-control/built-in-roles).
-
-<aside class="note warning"> 
-
-<h1> Warning </h1> 
+Identities are used on the management cluster and the VMs/clusters/workloads which get provisioned by the management cluster.
 
-The capability to set credentials using environment variables has been removed, the required approach is to use `AzureClusterIdentity` as explained [here](multitenancy.md) 
```

Comment:
Should we be keeping this line? I think previous users may be confused if their credentials no longer work via environment variables. I think it can be valuable to also mention that pod identity is deprecated somewhere on this page.